bravado/http_future.py

```diff
@@ -143,12 +143,13 @@ def response(self, timeout=None, fallback_result=None, exceptions_to_catch=FALLB
             if end_time is None:
                 end_time = monotonic.monotonic()
             exc_info = sys.exc_info()
-            if fallback_result:
+            if self.operation and not self.operation.swagger_spec.config['bravado'].disable_fallback_results:
                 swagger_result = fallback_result(e)
             else:
                 six.reraise(*exc_info)
 
-        response_metadata = BravadoResponseMetadata(
+        metadata_class = get_metadata_class(self.operation.swagger_spec.config['bravado'].response_metadata_class)
```

Comment:
I would refactor a bit the code to have the metadata class saved into the config object.

Doing so allows to have the class type (not string) in the config namedtuple and you don't have to call `get_metadata_class` for each request